services/requester/requester.go

```diff
@@ -53,6 +54,8 @@ var (
 const minFlowBalance = 2
 const coaFundingBalance = minFlowBalance - 1
 
+const LatestBlockHeight uint64 = math.MaxUint64 - 1
```

Comment:
I think it would be better to move this to models package where we have defined all the types

Reply:
I tried that first, but the `models.go` file is under the `api` package, which currently creates an import cycle and errors out. In a follow-up PR I will try to further split up the `api` package into more cohesive ones.